chalice/local.py

```diff
@@ -161,6 +162,22 @@ def _is_binary(self, headers):
         # type: (Dict[str,Any]) -> bool
         return headers.get('content-type', '') in self._binary_types
 
+    def _get_local_context(self, headers):
+        # type: (Dict[str,Any]) -> bool
```

Comment:
This is causing the tests to fail. Looking at your changes in here and in `create_lambda_event()` it seems you should be using `# type: (Dict[str, str])) -> Dict[str, str]`.

The `headers` passed into `create_lambda_event()` are `Dict[str, str])`. You are returning a collection that can be used to replace the existing `headers`.